### DISCUSSION TITLE
Microsoft passwordless account

### DISCUSSION THREAD

**Discussion by Swallowtail23** [abraunegg/onedrive#1632] [Q&A] [answered]
Any issues to existing running / synced instances if the Microsoft account in use is switched to be passwordless, per recent changes available with Microsoft accounts?

**Comment by abraunegg** [comment]
@Swallowtail23 
> Any issues to existing running / synced instances if the Microsoft account in use is switched to be passwordless, per recent changes available with Microsoft accounts?

Most likely the existing running process will fail with authentication, requiring a new authentication token to be submitted to the application to use.

If you can test and report back your experience that would be greatly appreciated.

**Comment by Swallowtail23** [comment, marked as answer]
Well it all re-auth'ed OK, 5 x servers now doing a resync - but with 1.5TB of data that will take a while to scan through. I'll confirm once it's done, but so far so good. I'm still on 2.4.11 btw, complied from source on Centos boxes.

**Comment by Swallowtail23** [comment]
All seems ok...
